### PR TITLE
Tab Improvements

### PR DESCRIPTION
**Pull request by ryanjwilke**
Improved some of the tab styles to make them feel a little more responsive and interactive. Also made a few adjustments to the React component to help improve the CSS that's being output as well.

### CODE REVIEW

File: package/src/tabs/tabs.tsx [hunk 1]
```diff
@@ -46,29 +56,41 @@ export const Tabs: React.FunctionComponent<ITabs> = ({
...
   }),[])
+
+  let linkClasses = classnames('tab-link', `tab-link-${type}`, `tab-link-${type}-${color}`)
+
   return (
     <div className={classes}>
       <div className={labelsClasses}>
-        {tabs.map((t) => <div className={`tab-label ${React.isValidElement(t) && t.key && activeKey === t.props.tabKey ? "tab-active" : ""}`} key={`tab-${React.isValidElement(t) && t.key}`} onClick={() => onLabelClick(t)}>{React.isValidElement(t) && t.props.label}</div>)}
-      </div>
-      <div className="active-tab">
```
Comment (ryanjwilke) [inline package/src/tabs/tabs.tsx:60]: @mgiamberardino I couldn't quite figure out what this `.active-tab` was for. Is that where you're dropping content once you've clicked on a certain tab?

File: package/src/tabs/tabs.tsx [hunk 2]
```diff
@@ -7,24 +7,24 @@ export interface ITabs {
...
-  compact = false,
+  color = "jade",
   defaultActiveTab,
-  inline = false,
+  type = "default",
   onTabChange,
 }) => {
   const [activeKey, setActiveKey] = useState<string>();
-  const classes = useMemo(() => classnames(classNames, "tabs"), [classNames])
+  const containerClasses = classnames('tab-container', `tab-container-${type}`, classNames);
```
Comment (mgiamberardino) [inline package/src/tabs/tabs.tsx:27]: Why removing the `useMemo` here? We should use hooks to keep states inside the components.

**Review by mgiamberardino — CHANGES_REQUESTED** [review]
LGTM. Just one comment to avoid extra re-renders.

File: package/src/tabs/tabs.tsx (hunk 2 above)
Comment (ryanjwilke) [reply package/src/tabs/tabs.tsx]: That should be fixed now.

File: package/src/tabs/tabs.tsx [hunk 3]
```diff
@@ -46,29 +46,41 @@ export const Tabs: React.FunctionComponent<ITabs> = ({
...
+
   return (
-    <div className={classes}>
-      <div className={labelsClasses}>
-        {tabs.map((t) => <div className={`tab-label ${React.isValidElement(t) && t.key && activeKey === t.props.tabKey ? "tab-active" : ""}`} key={`tab-${React.isValidElement(t) && t.key}`} onClick={() => onLabelClick(t)}>{React.isValidElement(t) && t.props.label}</div>)}
+    <div className={containerClasses}>
+      <div className={listClasses}>
+        {tabs.map((t) => {
+          if (!React.isValidElement(t)) return
+          const linkClasses = classnames({
```
Comment (ryanjwilke) [inline package/src/tabs/tabs.tsx:61]: @mgiamberardino Would you use the `useMemo` function here as well?

**Review by mgiamberardino — APPROVED** [review]
LGTM

**Pull request merged**